src/oscar/core/loading.py

```diff
@@ -239,7 +239,7 @@ def get_model(app_label, model_name):
             warnings.warn(
                 'Model %s has recently moved from %s to the application %s, '
                 'please update your imports.' % (model_name, original_app_label, app_label),
-                RemovedInOscar31Warning, stacklevel=2)
+                RemovedInOscar32Warning, stacklevel=2)
```

Comment:
The whole `if oscar_moved_model` block this is in, was [added](https://github.com/django-oscar/django-oscar/commit/2af6753c6733198d66519becd78e62f8af0b2cf0) with the `communication` app. The affected models were [removed](https://github.com/django-oscar/django-oscar/commit/c30de9d008bb11ae80e230f156111726b4c24d00) from `MOVED_MODELS` with the release of 3.0 alpha, but this code was left behind. Maybe we should remove it completely?

Reply:
We discussed this at the time and decided to leave this code here in case we need to move other models in future.